### PR TITLE
docs: update community related docs

### PR DESCRIPTION
**Pull request by alessfg**
Notable differences:
- Add a PR template
- Update Code of Conduct
- Add Contributing link and Copyright notice to README
- Remove Docker pulls vanity metric (vanity metrics are meh) from README
- Minor updates throughout all the community related docs to bring the docs up to speed with the latest community doc guidelines

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -22,20 +14,24 @@ This isn't the right place to get support for NGINX specific questions, but the
...
+For a comprehensive list of administration and deployment guides for all NGINX products, check out <https://docs.nginx.com>.
 
 ### Mailing List
 
 Want to get in touch with the NGINX development team directly? Try using the relevant mailing list found at <https://mailman.nginx.org/mailman3/lists/>!
 
 ## Contributing
 
-Please see the [contributing guide](CONTRIBUTING.md) for guidelines on how to best contribute to this project.
+Please see the [contributing guide](https://github.com/nginxinc/nginx-s3-gateway/blob/main/CONTRIBUTING.md) for guidelines on how to best contribute to this project.
```
Comment (4141done) [inline SUPPORT.md:33]: nit: We should stick to relative links. Part of the workflow to contribute is to clone the repo which would make these links point at the wrong place.  GitHub recommends this as well https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

**Review by 4141done — APPROVED** [review]
👍 🐳 👍

File: SUPPORT.md (hunk 1 above)
Comment (alessfg) [reply SUPPORT.md:33]: I have been hurt in a few places by using relative links, but I guess those places might be the exception to the rule and not the norm. Given GH recommends them as the standard, I am happy to revert back to relative links :)

**Comment by alessfg** [comment]
I'll change the full links back to relative links and merge the PR 😄

**Pull request merged**